src/index.html

```diff
@@ -0,0 +1,828 @@
+              </path>
+            </g>
+          </svg>
+          <span class="copyright">© 2019 NEWSWEEK</span>
+        </div>
+        <div class="col-lg-4 col-md-12 col-sm-12 social col-12">
+          <i class="fab fa-facebook-f"></i>
+          <i class="fab fa-twitter-square"></i>
+          <i class="fab fa-instagram"></i>
+        </div>
+      </div>
+      <div class="row second-row no-gutters mt-2">
+        <div class="col-lg-7 col-md-12 col-sm-12 links col-12">
+          <span>About Us</span>
+          <span>Corrections</span>
+          <span>Contact Us</span>
+          <span>Editorial Guidelines</span>
+          <span>Advertise</span>
+          <span>Copyright</span>
+          <span>Terms & Conditions</span>
+          <span>Privacy Policy</span>
+          <span>Cookie Policy</span>
+          <span>Terms of Sale</span>
+          <span>Archive</span>
+          <span>Announcements</span>
+          <span>Consent preferences</span>
+        </div>
+        <div class="col-lg-4 offset-lg-1 col-md-12 col-sm-12 links col-12">
+          <span><b>Editions:</b></span>
+          <span>U.S. Edition</span>
+          <span>日本</span>
+          <span>한국</span>
+          <span>Pakistan</span>
+          <span>Polska</span>
+          <span>România</span>
+        </div>
+      </div>
+    </footer>
+  </main>
```

Comment:
The `<footer>` tag shouldn't be inside the `<main>` tag each one of represents a different part of the website content.

Reply:
Thanks for the feedback Redvanisation, I'll read some more about semantic HTML.